examples/HydroValleys/case3.jl

```diff
@@ -81,19 +70,4 @@ using Test
 #' Bound
 @test SDDP.calculate_bound(m.forward_graph) >= 1.1e4
 #' Number of Simulations
-@test length(results[:simulations]) == 100
-
-#' ## Plot Aggregated Results
-if plot_bool == true
-    HydroPowerModels.plot_aggregated_results(results)
-end
-
-#' # Annex 1: Case Summary
-if plot_bool == true
-    PowerModels.print_summary(alldata[1]["powersystem"])
-end
-
-#' # Annex 2: Plot Results
-if plot_bool == true
-    HydroPowerModels.plotresults(results)
-end
+@test length(results[:simulations]) == 100
```

Comment:
```suggestion
@test length(results[:simulations]) == 100

```

Uma boa pratica criar uma linha vazia no fim.